WordPress/src/main/res/values/available_languages.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--Warning: Auto-generated file, do not edit.-->
-<resources>
-  <string-array name="available_languages" translatable="false">
+<resources xmlns:tools="http://schemas.android.com/tools">
+  <string-array name="available_languages" translatable="false" tools:ignore="InconsistentArrays">
```

Comment:
Question (❓): Is ignoring `InconsistentArrays` needed for the `main` res related file too, I had the impression that this is only needed for the `jetpack` res related file. 🤔

Reply:
This might not, but the way our `release-toolkit` action that is in charge of creating this file works, it currently doesn't have a flag to tell it when to add this ignore or not, and simply adds it in all cases.

Which is why it's present in all instances of this file being created, i.e. both `main` and `jetpack` sets. I figured that while this might not be absolutly necessary on `main`, it doesn't hurt so it's ok to keep to avoid overcomplicating the fastlane action creating that file 🤷

Reply:
Yeap, yeap, if that's the case, then I am all-in for keeping both, thanks for the explanation Olivier! 👍 🙏